src/shogun/base/AnyParameter.h

```diff
@@ -213,7 +213,12 @@ namespace shogun
 		{
 			return m_properties;
 		}
-
+		using ParameterRange = std::variant<std::pair<Any, Any>, std::vector<Any>>;
```

Comment:
@vigsterkr  can I assume that type of std::pair<Any, Any> is float64_t/int and std::vector<Any> is kernel?